### PR TITLE
Add participations for Org-Roles.

### LINKED ISSUE #2107: Participation for org-role
Not only persons can participate in dossiers but also their `OrgRole`. E.g. Hugo Boss as CTO of Boss. Corp. This is most likely relevant when pre-selecting addresses.

### PR DESCRIPTION
**Pull request by deiferni**
This PR changes participations in a dossier by also enabling an `OrgRole` to participate in dossiers - in addition to `Contacts` (`Persons` and `Organizations`).

screen shot 2016-09-01 at 11 10 56

Closes #2107.

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross plz have a look

File: opengever/contact/tests/test_participation.py [hunk 1]
```diff
@@ -74,13 +74,49 @@ def test_update_roles_add_new_roles(self):
...
 
+class TestOrgRoleParticipation(unittest2.TestCase):
+
+    layer = MEMORY_DB_LAYER
+
+    def test_adding(self):
+        person = create(Builder('person').having(
+            firstname=u'peter', lastname=u'hans'))
+        organization = create(Builder('organization').named('ACME'))
+        orgrole = create(Builder('org_role').having(
```
Comment (phgross) [inline opengever/contact/tests/test_participation.py]: Use the `in_orgs` helper of the PersonBuilder.

File: opengever/contact/tests/test_participation.py [hunk 2]
```diff
@@ -74,13 +74,49 @@ def test_update_roles_add_new_roles(self):
...
+
+        participation = create(Builder('org_role_participation').having(
+            org_role=orgrole,
+            dossier_oguid=Oguid('foo', 1234)))
+
+    def test_participation_can_have_multiple_roles(self):
+        person = create(Builder('person').having(
+            firstname=u'peter', lastname=u'hans'))
+        organization = create(Builder('organization').named('ACME'))
+        orgrole = create(Builder('org_role').having(
```
Comment (phgross) [inline opengever/contact/tests/test_participation.py]: Same here

File: opengever/contact/browser/templates/participations.pt [hunk 3]
```diff
@@ -23,11 +22,11 @@
           </a>
         </li>
         <li>
-          <a class="remove-action"
-             tal:attributes="href string:${context/absolute_url}/${participation/wrapper_id}/remove"
-             i18n:translate="button_remove">
-            Remove
-          </a>
+        <a class="remove-action"
```
Comment (phgross) [inline opengever/contact/browser/templates/participations.pt]: -->

File: opengever/contact/tests/test_contact_unit.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from opengever.contact.models import Contact
+from opengever.testing import MEMORY_DB_LAYER
+import unittest2
+
+
+class TestArchivedURL(unittest2.TestCase):
```
Comment (phgross) [inline opengever/contact/tests/test_contact_unit.py]: Copy & Paste

I would propose using `TestPolymorphicQuery`, then you can shorten all the test methods.

File: opengever/contact/tests/test_contact_unit.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        query = Contact.query.polymorphic_by_searchable_text(
+            text_filters=['Pete*'])
+        self.assertEqual([peter, peter_co], query.all())
+
+    def test_polymorphic_returns_org_roles_with_matching_function_for_one_entity(self):
+        ftw = create(Builder('organization').named(u'4teamwork AG'))
+        meier_ag = create(Builder('organization').named(u'Meier AG'))
+
+        peter = create(Builder('person')
+                       .having(firstname=u'Peter', lastname=u'M\xfcller'))
```
Comment (phgross) [inline opengever/contact/tests/test_contact_unit.py]: Uses PersonBuilder `in_orgs` method

**Comment by phgross** [comment]
👍

File: opengever/contact/tests/test_participation.py (hunk 1 above)
Comment (deiferni) [inline opengever/contact/tests/test_participation.py]: i need a reference to the org_role later (for the participation) so i'd prefer to create it explicitly

File: opengever/contact/tests/test_participation.py (hunk 2 above)
Comment (deiferni) [inline opengever/contact/tests/test_participation.py]: here too 😉

File: opengever/contact/tests/test_contact_unit.py (hunk 5 above)
Comment (deiferni) [inline opengever/contact/tests/test_contact_unit.py]: i need the roles for assertions, so i'd rather not. see above 😉

**Comment by deiferni** [comment]
@phgross updated

**Pull request merged**